torch/onnx/_internal/exporter.py

```diff
@@ -879,8 +936,7 @@ def _assert_fake_tensor_mode(self):
         if (
             has_any_fake_tensor or has_any_fake_param_or_buffer
         ) and not self.options.fake_context:
-            return OnnxExporterError(
-                self.options.diagnostic_context,
+            return RuntimeError(
```

Comment:
There was a bug using "return" instead of "raise". I cannot correct it as `test_fx_symbolic_tracer_large_scale_exporter` tests start to fail if I do. @thiagocrepaldi please take a look later.

Reply:
Please create an issue and assigne me or maybe @titaiwangms who will help with some fake tasks